### PR TITLE
Misc doc updates

### PR DESCRIPTION
**Pull request by aspect**
Mostly small README changes and updated links for external guides.

### CODE REVIEW

File: wasm/README.md [hunk 1]
```diff
@@ -46,13 +39,43 @@ WebSocket implementation. Two of such modules are [WebSocket](https://www.npmjs.
...
+## Loading in a Node.js App
+
+```javascript
+// W3C WebSocket module shim
 globalThis.WebSocket = require('websocket').w3cwebsocket;
-// `isomorphic-ws` module
-globalThis.WebSocket = require('isomorphic-ws');
+
+let {RpcClient,Encoding,init_console_panic_hook,defer} = require('./kaspa-rpc');
+// init_console_panic_hook();
```
Comment (coderofstuff) [inline wasm/README.md:66]: Is this intended to be commented out? If it is, you should say when it should be uncommented.

File: wasm/README.md [hunk 2]
```diff
@@ -46,13 +39,43 @@ WebSocket implementation. Two of such modules are [WebSocket](https://www.npmjs.
...
+
+let rpc = new RpcClient(Encoding.Borsh,"ws://127.0.0.1:17110");
+
+(async () => {
+    await rpc.connect();
+
+    let info = await rpc.getInfo();
+    console.log(info);
+
+    await rpc.disconnect();
```
Comment (coderofstuff) [inline wasm/README.md:76]: Optional, just to follow some good practices even in examples.

```suggestion
    try {
        let info = await rpc.getInfo();
        console.log(info);
    } finally {
        await rpc.disconnect();
    }
```

**Review by coderofstuff — APPROVED** [review]
My comments are optional. Changes and links are accurate and resolve to correct locations.

File: wasm/README.md (hunk 1 above)
Comment (aspect) [reply wasm/README.md]: Thanks for noting this. This was written months ago and I completely overlooked that functions have been renamed to `initConsolePanicHook()` to have camel case.  Updated, added comments and added `initBrowserPanicHook()` into the mix.

File: wasm/README.md (hunk 2 above)
Comment (aspect) [reply wasm/README.md]: Added try/finally

**Review by coderofstuff — APPROVED** [review]

**Review by michaelsutton — APPROVED** [review]

**Pull request merged**